src/styles/generic/_shared.scss

```diff
@@ -26,6 +27,10 @@
   outline: none !important;
 }
 
+// https://css-tricks.com/fixed-headers-on-page-links-and-overlapping-content-oh-my/
+html {
+  scroll-padding-top: calc(#{$TOPNAV_HEIGHT} + 24px);
```

Comment:
24 was arbitrary but it felt nice to give it a tad more space. otherwise the text is right up against the bottom of the header.